src/components/Application/Navigation.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
+import { Button, Segment, Sticky } from 'semantic-ui-react'
+import {
+  MethodRevalidate,
+  MethodToCallProps,
+  SectionAndPage,
+  SectionDetails,
+  SectionsStructureNEW,
+} from '../../utils/types'
+import strings from '../../utils/constants'
+import { useRouter } from '../../utils/hooks/useRouter'
+
+interface NavigationProps {
+  current: SectionAndPage
+  sections: SectionsStructureNEW
+  serialNumber: string
+  requestRevalidation: MethodRevalidate
+}
+
+const Navigation: React.FC<NavigationProps> = ({
+  current,
+  sections,
+  serialNumber,
+  requestRevalidation,
+}) => {
+  const { push } = useRouter()
+
+  const currentSectionDetails = sections[current.sectionCode].details
+
+  const hasNextSection = Object.values(sections).find(
+    ({ details: { index } }) => index > currentSectionDetails.index
+  )
+  const hasPreviousSection = Object.values(sections).find(
```

Comment:
these ones return previous or next section || null, so they can be named as previousSeciton and nextSection and used in getPreviousSecitonPage or getNextSectionPage

Reply:
Yep, good point.

Reply:
Done: 7de0277